apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomFissClaimSourceTest.java

```diff
@@ -34,11 +34,11 @@ public void threeMaxToReturn() throws Exception {
 
     assertEquals(true, source.hasNext());
     claim = source.next();
-    assertEquals("12793", claim.getDcn());
+    assertTrue(claim.getDcn().length() > 0);
```

Comment:
assuming this change reflects the 'randomness'  of the claims genearation.

Reply:
Correct.  The big thing being tested here is just that exactly three claims were returned.  Having specific value checks for Dcn there just made the tests brittle without adding any benefit.